x/btccheckpoint/keeper/msg_server.go

```diff
@@ -92,6 +92,11 @@ func (ms msgServer) InsertBTCSpvProof(ctx context.Context, req *types.MsgInsertB
 		return nil, err
 	}
 
+	// At this point, the BTC checkpoint is considered the first valid submission
```

Comment:
this is not exactly true as it may be as well not first submission. If the intention is to refund only first one, the place to do so would be elsewere.

On the other hand, I thought we want to refund all valid submissions ?

Reply:
Yeah this is a typo we want to refund all valid submissions (more specifically, every unique pair of BTC txs). Will clarify in a follow-up